doc/release_notes.rst

```diff
@@ -28,8 +36,6 @@ This part of documentation collects descriptive release notes to capture the mai
 
 * Boost computational performances of fix_overpassing_line `PR #807 <https://github.com/pypsa-meets-earth/pypsa-earth/pull/807>`__
 
-**Minor Changes and bug-fixing**
-
```

Comment:
This has been dropped by mistake probably, can you restore it?

Reply:
Actually, I haven't been sure if we can't count them as major improvements 🙂 FT-reimplementation and hosting gadm files change the interfaces (which makes them major by definition) and are also great improvement. By since the authors have qualified them as minor, let's restore probably them 😄